frosh/code-quality-meta/0.3/root/phpstan.neon.dist

```diff
@@ -2,7 +2,9 @@ parameters:
     level: 8
     tmpDir: var/cache/phpstan_dev
     inferPrivatePropertyTypeFromConstructor: true
-    checkMissingIterableValueType: false
+    ignoreErrors:
+    		-
```

Comment:
the line indent seems wrong. previously was only whitespace now mixed with tab

Reply:
Fixed,